### PR TITLE
Live table index.html is a valid HTML 5 file now

### PR DESCRIPTION
**Pull request by exarus**

### CODE REVIEW

File: LiveTable/index.html [hunk 1]
```diff
@@ -49,5 +49,3 @@
   };
 
 </script>
-</body>
```
Comment (tshemsedinov) [inline LiveTable/index.html:52]: Why `</body></html>` removed?

File: LiveTable/index.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html>
-<body>
```
Comment (tshemsedinov) [inline LiveTable/index.html:3]: Where is body?

**Comment by exarus** [comment]
Trying to keep code as simple as possible. Reader should focus JavaScript, not HTML.
According to HTML 5 specification:

> A **body** element's start tag may be omitted if the element is empty, or if the first thing inside the **body** element is not a space character or a comment, except if the first thing inside the body element is a **meta**, **link**, **script**, **style**, or **template** element.
> A **body** element's end tag may be omitted if the **body** element is not immediately followed by a comment.
> An **html** element's end tag may be omitted if the **html** element is not immediately followed by a comment.

https://www.w3.org/TR/html5/syntax.html#optional-tags
Any of popular HTML5-compatible browsers don't have any problems with closing tags.

**Comment by exarus** [comment]
Any updates on this?

**Review by tshemsedinov — APPROVED** [review]

**Pull request merged**